js/dropdowns.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-/* global require, module, document */
-
 var $ = require('jquery');
-var perfectScrollbar = require('perfect-scrollbar/jquery')($);
+require('perfect-scrollbar/jquery')($);
```

Comment:
It's the other way around--we initialize the perfect-scrollbar plugin by passing it the global jQuery object. Incidentally, use of this library predates me on the project, so @noahmanger can field questions about what it actually does. The only thing I'm doing here is taking out an unnecessary variable declaration that was throwing a jshint error.